platforms/documentation/docs/src/docs/userguide/authoring-builds/tasks/more_about_tasks.adoc

```diff
@@ -81,180 +69,161 @@ include::sample[dir="snippets/tasks/defineAsKotlinDelegatedProperty/groovy",file
+This avoids duplication of code and minimizes redundancy:
 
-.Accessing tasks by their type
 ====
 include::sample[dir="snippets/tasks/accessUsingType/kotlin",files="build.gradle.kts[]"]
 include::sample[dir="snippets/tasks/accessUsingType/groovy",files="build.gradle[]"]
 ====
 
-WARNING: The following shows how to access a task by path.
-This is not a recommended practice anymore as it breaks <<task_configuration_avoidance#task_configuration_avoidance,task configuration avoidance>> and project isolation.
-Dependencies between projects <<declaring_dependencies_between_subprojects.adoc#declaring_dependencies_between_subprojects,should be declared as dependencies>>.
-
-You can access tasks from any project using the task's path using the `tasks.getByPath()` method.
-You can call the `getByPath()` method with a task name, or a relative path, or an absolute path.
-
-.Accessing tasks by path
-====
-include::sample[dir="snippets/tasks/accessUsingPath/kotlin",files="project-a/build.gradle.kts[];build.gradle.kts[]"]
-include::sample[dir="snippets/tasks/accessUsingPath/groovy",files="project-a/build.gradle[];build.gradle[]"]
-
-.Output of **`gradle -q hello`**
-----
-> gradle -q hello
-include::{snippetsPath}/tasks/accessUsingPath/tests/accessUsingPath.out[]
-----
-====
-
-Have a look at link:{javadocPath}/org/gradle/api/tasks/TaskContainer.html[TaskContainer] for more options for locating tasks.
-
 [[sec:configuring_tasks]]
 == Configuring tasks
 
-As an example, let's look at the `Copy` task provided by Gradle.
-To register a `Copy` task for your build, you can declare in your build script:
+Let’s look at the `Copy` task provided by Gradle as an example.
+To register a `Copy` task for your build, you can declare it in your build script:
 
-.Registering a copy task
 ====
 include::sample[dir="snippets/tasks/configureUsingBlock/kotlin",files="build.gradle.kts[tags=declare-task]"]
 include::sample[dir="snippets/tasks/configureUsingBlock/groovy",files="build.gradle[tags=declare-task]"]
 ====
 
 This registers a copy task with no default behavior.
-The task can be configured using its API (see link:{groovyDslPath}/org.gradle.api.tasks.Copy.html[Copy]).
-The following examples show several different ways to achieve the same configuration.
+The name of this task is `myCopy`, but it is of _type_ `Copy`.
+
+As such, the task can be configured using its API (see link:{groovyDslPath}/org.gradle.api.tasks.Copy.html[Copy]).
+
+TIP: You can have multiple tasks of the same _type_, but with different names.
+
+The following examples show several ways to achieve the same configuration.
 
-Just to be clear, realize that the name of this task is `myCopy`, but it is of _type_ `Copy`.
-You can have multiple tasks of the same _type_, but with different names.
-You'll find this gives you a lot of power to implement cross-cutting concerns across all tasks of a particular type.
+=== 1. Using the `named()` method:
```

Comment:
🤔 Ideally this should cover the newly added `named(Action)` that is presented in the release notes for 8.6
It can be added later though

Reply:
I will see if I can add this now. It might have to be added later.